website/docs/tutorials/08-cairo-1-support/03-cheatcodes/call.md

```diff
@@ -0,0 +1,47 @@
+# `call`
+
+```cairo
+extern fn call(contract: felt252, entry_point_selector: felt252, calldata: Array::<felt252>) -> Result::<(Array::<felt252>), felt252> nopanic;
+```
+
+Calls a [deployed](./deploy.md) contract. Unlike [invoke](./invoke.md), it **does not** mutate the blockchain state.
+
+- `contract` - deployed contract address
+- `entry_point_selector` - the name of the function you wish to call
+- `calldata` - arguments to the target function
+
+```cairo title="Example"
+use array::ArrayTrait;
+use result::ResultTrait;
+
+#[test]
+fn test_call_simple() {
+    let class_hash = declare('simple').unwrap();
```

Comment:
We should provide the contract alongside the example

Reply:
Sorry but I disagree, this code should be relatively short and show only how to use this function. Maybe this example is already too long and should be only:
```
let mut calldata = ArrayTrait::new();
calldata.append(3);
calldata.append(2);
calldata.append(5);
let return_data2 = call(deployed_contract_address, 'perform', calldata).unwrap();
assert(*return_data2.at(0_u32) == 25, 'check call result');
```
?
If anything's unclear, there's a link below to the full example so people can see everything around, I'd keep the docs brief.
If you have a strong opinion going in the opposite direction, please let me know.